Analysis/backgrounds_zgwg.tex

```diff
@@ -70,6 +71,7 @@ \subsection{Simulation of \vg\ Processes}
   }
   \label{fig:ewk_correction}
 \end{figure}
+\PH{Its not completely obvious which unceratiny in the plot maps to the uncertainty in the text, please fix lables or add more to caption}
```

Comment:
specified that shape and slope refer to the two Sudakov uncertainties mentioned later in the text.